examples/classification_sample/CMakeLists.txt

```diff
@@ -4,8 +4,8 @@
 # SPDX-License-Identifier: Apache-2.0
 # ******************************************************************************
 
-include_directories(${TensorFlow_INCLUDE_DIR})
-include_directories(${TensorFlow_INCLUDE_DIR}/external/nsync/public)
+# include_directories(${TensorFlow_INCLUDE_DIR})
```

Comment:
Can we delete these commented out lines?

Reply:
Done